dynamic/Kadane.go

```diff
@@ -0,0 +1,56 @@
+package main
+import "fmt"
+/**
+ * Program to implement Kadane’s Algorithm to calculate maximum contiguous subarray sum of an array
+ * Time Complexity: O(n)
+ *
+ * @author Ritik Agarwal
+*/
+ /**
+   * This method implements Max operation
+   *
+   * @param a,b Two integers
+   * @return The maximum among the two integers
+   */
+func max(a int, b int) int{
```

Comment:
there is an abundance these functions everywhere - use the standard library if you can't find the max function defined in the repo. You can search the repo for `Max` and it will show you the implementation, so just import it instead of reinventing the wheel.